pep/app/pods/components/page/nav/component.ts

```diff
@@ -37,6 +49,25 @@ export default class PageNav extends Component<PageNavArgs> {
         return this.auth.openLoginModal(true);
     }
 
+    /**
+     * Open the about modal dialog
+     *
+     * @returns Promise<Void>
+     * @memberof PageDrawer
+     */
+    @action
+    async openAboutModal() {
+        try {
+            const result = await this.ajax.request<ServerStatus>('Session/Status');
+            return this.modal.open('user/about', {
+                serverInformation: result,
+                clientBuildVersion: ENV.buildVersion
+            });
+        } catch (error) {
+            return error;
```

Comment:
i think you'd want to `throw` the error here instead of resolving the process successfully with an error. Or in this case you could remove the `try/catch` altogether, since this is the default behavior of async functions (unless you add `loadingBar` in which case you'd call `loadingBar.hide()` in a `finally` block.

Reply:
ideally, you could also use `this.notifications.error(...)` to display an error toast in the catch, in case we couldnt retrieve the server info for w/e reason.